lib/cadet/assessments/assessments.ex

```diff
@@ -481,6 +499,33 @@ defmodule Cadet.Assessments do
     Question.changeset(%Question{}, params_with_assessment_id)
   end
 
+  def insert_voting(voting_params, assessment_id) do
+    question =
+      Question
+      |> where(assessment_id: ^assessment_id)
+      # voting assessment only has one question
```

Comment:
Can we get rid of this assumption? Make this function operate on the question ID?